$MirOS: ports/www/lynx/patches/patch-makefile_in,v 1.3 2008/03/09 18:19:43 tg Exp $

	BSD make specific. Portable would be
	+ LY_CFLAGS='$(CFLAGS)'
	or something like that, but :Q also quotes the quotes.

--- makefile.in.orig	Sun Sep 21 22:49:24 2008
+++ makefile.in	Wed Dec 17 00:14:05 2008
@@ -175,13 +175,13 @@ SITE_DEFS = # Your defines here
 # for installation of local execution links, please see the file userdefs.h
 
 WWW_CFLAGS = \
-	LY_CFLAGS="$(CFLAGS)" \
-	CPPFLAGS="$(CPPFLAGS)" \
+	LY_CFLAGS=${CFLAGS:Q} \
+	CPPFLAGS=${CPPFLAGS:Q} \
 	LYFLAGS="$(SITE_LYDEFS)"
 
 SRC_CFLAGS = \
-	CFLAGS="$(CFLAGS)" \
-	CPPFLAGS="$(CPPFLAGS)" \
+	CFLAGS=${CFLAGS:Q} \
+	CPPFLAGS=${CPPFLAGS:Q} \
 	LIBS="$(LIBS) $(RESOLVLIB) $(WAISLIB) $(SITE_LIBS)" \
 	SITE_DEFS="$(SITE_DEFS)" \
 	WWWINC=$(WWW_DIR) \
